showcase/app/templates/components/form/index.hbs

```diff
@@ -0,0 +1,1078 @@
+                <F.Error>This is a field-level error</F.Error>
+              </Hds::Form::TextInput::Field>
+            </FORM.SectionFieldGroup>
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label="with two errors">
+        <Hds::Form as |FORM|>
+          <FORM.Section>
+            <FORM.SectionFieldGroup>
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>Control 1</F.Label>
+                <F.HelperText>Lorem ipsum dolor sit amet consectetur.</F.HelperText>
+              </Hds::Form::TextInput::Field>
+
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>Control 2</F.Label>
+                <F.Error>This is a field-level error.</F.Error>
+                <F.Error>Yet another error example for a total of two.</F.Error>
+              </Hds::Form::TextInput::Field>
+            </FORM.SectionFieldGroup>
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label=" with 3 fields of equal width">
+        <Hds::Form as |FORM|>
+          <FORM.Section>
+            <FORM.SectionFieldGroup>
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>Control 1</F.Label>
+              </Hds::Form::TextInput::Field>
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>Control 2</F.Label>
+                <F.HelperText>Lorem ipsum dolor sit amet consectetur.</F.HelperText>
+              </Hds::Form::TextInput::Field>
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>Control 3</F.Label>
+              </Hds::Form::TextInput::Field>
+            </FORM.SectionFieldGroup>
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label="3 fields, mix of widths (1/3, auto, & 6em)">
+        <Hds::Form as |FORM|>
+          <FORM.Section>
+            <FORM.SectionFieldGroup as |FG|>
+              <Hds::Form::TextInput::Field as |F|>
+                <F.Label>City</F.Label>
+              </Hds::Form::TextInput::Field>
+
+              <FG.Item @basis="auto">
+                <Hds::Form::Select::Field as |F|>
+                  <F.Label>State</F.Label>
+                  <F.Options>
+                    <option value="state-1">Ohio</option>
+                    <option value="state-2">Massachusetts</option>
+                    <option value="state-3">Washington</option>
+                    <option value="state-4">Florida</option>
+                    <option value="state-4">North Carolina</option>
+                  </F.Options>
+                </Hds::Form::Select::Field>
+              </FG.Item>
+
+              <Hds::Form::TextInput::Field @width="6em" as |F|>
+                <F.Label>Zip</F.Label>
+              </Hds::Form::TextInput::Field>
+            </FORM.SectionFieldGroup>
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label="direction=column">
```

Comment:
[note] re. the direction=column, see the comment I've left here:  https://docs.google.com/document/d/1H92eLJQxBl7QObrpUTNgno7YGgvTzZkTQnl3UF6oO5k/edit?disco=AAABlnVN2S4